services/121-service/src/payments/fsp-integration/intersolve-visa/intersolve-visa.service.ts

```diff
@@ -567,7 +554,7 @@ export class IntersolveVisaService
         ))
     ) {
       await this.intersolveVisaChildWalletScopedRepository.updateUnscoped(
-        { tokenCode: tokenCode ?? undefined },
+        { tokenCode: tokenCode },
```

Comment:
?? undefined vanished? Should be there because of strict null check?

Reply:
It was not needed anymore since that function is only called when the variable is a string